settings.gradle

```diff
@@ -5,5 +5,5 @@ include 'buildSrc'
 include 'frontend-service-client'
 include 'frontend-service'
 include 'frontend-infrastructure'
-include 'data-service-model'
+include 'data-service-client'
```

Comment:
Oh, this explains the Client class above. Why the change from model to client here? I thought the idea was to keep the model objects/DAOs in a package on their own, so that the server won't have to depend on the code the implements the client, and vice versa.